RockPaperScissors/control.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra/base'
+class RockPaperScissors < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    erb :index
+  end
+
+  post '/result' do
+    @player_1_name = params[:player_1_name]
+    @player_2_name = params[:player_2_name]
+    erb :play
```

Comment:
you can print the params to see what the structure is
`p params`
 
Then get the weapon choices from the params.  It could be something like this (depending on params structure).. but you might need to do this differently for radio buttons.
```
@player1_weapon = params[:player_1_weapon]
@player2_weapon = params[:player2_weapon]
```
then do your game logic, perhaps in the controller at first